README.md

```diff
@@ -59,6 +59,8 @@ sudo apt-get install -y libcairo2-dev libpango1.0-dev libpng-dev libjpeg-dev lib
    ```
 4. Close the privileged PowerShell prompt.
 
+[Install Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10). Rancher Desktop on Windows requires this.
```

Comment:
Once issue 185 is merged in this will be done by `windows-setup.ps1`, if wsl is not installed.

Reply:
@ericpromislow will it be installed for dev environments or just when you run the app?

Reply:
It will be done when `windows-setup.ps1` is run, so this is for devs.

Hmm, I don't have a solution for users yet, because they don't run `windows-setup.ps1`, right?